Sources/BraintreeCoreSwift/BTClientToken.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+
+/// An authorization string used to initialize the Braintree SDK
+@objcMembers public class BTClientToken: NSObject, NSCoding, NSCopying {
+
+    // MARK: - Public Properties
+
+    /// The client token as a BTJSON object
+    public let json: BTJSON
```

Comment:
Shall we add a TODO to revisit the access level of these properties?

Reply:
Good call, added a TODO for this, looks like most can be internal once BTAPIClient + BTAnalyticsService are converted